concepts/0003-protocols/roles-participants-etc.md

```diff
@@ -1,83 +1,41 @@
-# Roles, Participants, Parties, and Controllers
+### Roles, Participants, Parties, and Controllers
 
-## Roles
+#### Roles
 
-The __roles__ in a protocol are the perspectives (responsibilities, privileges) that parties take on an
-interaction. 
+The __roles__ in a protocol are the perspectives (responsibilities, privileges) that parties take i an
+interaction.
 
 This perspective is manifested in three general ways:
 
  * by the expectations that a party takes on in a protocol (ex. a role may be expected to do something to start a protocol).
  * by the messages that a party can and does use in the course of the protocol (some messages may be reserved for a single role, while other may used by some if not all roles).
  * by the state and the transition rules
- 
-Like parties, roles are normally known at the start of the protocol but this is not a requirement.
 
-In an auction protocol, there are only two roles--*auctioneer*
-and *bidder*--even though there may be many parties involved.
+Like parties, roles are normally known at the start of the protocol but this is not a requirement.
 
-## Participants
+#### Participants
 
-The __participants__ in a protocol are the agents that consume and/or emit
```

Comment:
I don't like losing this example. I have had several conversations with people who don't get the subtlety that's being described here.

Reply:
I think this example creates far more confusion about protocols, agents and DIDComm than it clarifies.  How are all those devices connected?  Where do the keys reside?  How do messages get to the different devices?  What about when the phones aren't close enough for Bluetooth?   IMHO, this is far too complex an example to be included in this intro.

What specific point is the example attempting to convey?

Reply:
The point is that agents who don't receive and emit messages are not participants, even if they belong to one of the parties that's interacting. What's at issue here is the definition of "participant" -- and we can't define it without explaining it accurately.

Reply:
Reverted